Views/Shared/_NavbarView.cshtml

```diff
@@ -14,7 +14,7 @@
         <div class="navbar-collapse collapse">
             <ul class="nav navbar-nav navbar-right">
                 <li><a asp-area="" asp-controller="User" asp-action="New">Create an Account</a></li>
-                <li><a asp-area="" asp-controller="Products" asp-action="New">Add Product</a></li>
+                <li><a asp-area="" asp-controller="Products" asp-action="New">Sell a Product</a></li>
```

Comment:
@MattKraatz I second @Anulfo.  The `asp-action` needs to be changed to `Create`.  Otherwise LGTM.